manifests/install/client.pp

```diff
@@ -2,8 +2,6 @@
 #
 # Install the sssd-client package
 #
-class sssd::install::client (
-  $ensure = 'latest'
-){
-  package { 'sssd-client': ensure => $ensure }
+class sssd::install::client {
```

Comment:
This new rendition needs to have `assert_private()`. The old one could be used standalone if you so chose.